### DISCUSSION TITLE
企鹅物流上传问题

### DISCUSSION THREAD

**Discussion by QCLY99** [MaaAssistantArknights/MaaAssistantArknights#6237] [Q&A]
我这边刷理智的时候出现"UnknownStage，放弃上传企鹅物流"的字样，但我在高级设置中填入了企鹅物流的数字ID的，不知道是什么情况，请问如何解决呢？

![屏幕截图 2023-06-27 183415](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/144203096/204856ba-80c8-4eef-993b-9017eff09ef6)

**Comment by ABA2396** [comment]
活动图的上报要等地图资源更新，如果更新了的话会在版本更新的日志里说的
